tesseract_ros/tesseract_examples/tesseract_ros_examples/src/puzzle_piece_plan.cpp

```diff
@@ -27,11 +27,14 @@
 TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <ros/ros.h>
 #include <fstream>
+#include <tesseract_msgs/ModifyEnvironment.h>
```

Comment:
Running this example I get the error 
```
Joint groups are currently not supported by inverse kinematics!
```

Also, plotting is very slow to start on this. It takes like 30 seconds.

Reply:
This is expected at the moment. Currently do not have an implementation for inverse kinematics for a tree structure.

Reply:
I noticed the slow to start. The very first message it needs to create all of the ogre entities for the trajectory for visualization which is the reason for the slow start. I need to profile it to see what the bottle neck is.